source/tests/pt/model/test_dp_hessian_model.py

```
@@ -0,0 +1,186 @@
+# SPDX-License-Identifier: LGPL-3.0-or-later
+import unittest
+
+import numpy as np
+import torch
```

Ruff

`torch` imported but unused

Remove unused import: `torch`

(F401)